src/components/passwordReset/RequestResetPasswordModal.tsx

```diff
@@ -0,0 +1,128 @@
+import { ForgotPasswordDto } from '@FullStackMap/from-a2b';
+import { Button, Modal, Space, TextInput } from '@mantine/core';
+import { useForm } from '@mantine/form';
+import { useDisclosure } from '@mantine/hooks';
+import { notifications } from '@mantine/notifications';
+import { IconCheck, IconX } from '@tabler/icons-react';
+import { zodResolver } from 'mantine-form-zod-resolver';
+import { z } from 'zod';
+import { AnoAuthController } from '../../services/BaseApi';
+
+export interface RequestResetPasswordModalProps {
+  isOpen: boolean;
+  close: () => void;
+}
+
+export const RequestResetPasswordModal = (
+  props: RequestResetPasswordModalProps,
+) => {
+  const [
+    isRequestResetPasswordButtonLoading,
+    {
+      toggle: toggleRequestResetPasswordLoading,
+      close: disableRequestResetPasswordLoading,
+    },
+  ] = useDisclosure(false);
+
+  const resetPasswordRequestSchema = z.object({
+    email: z.string().email('Un email valide est requis'),
+  });
+
+  const resetPasswordRequestForm = useForm({
+    validateInputOnChange: true,
+    initialValues: {
+      email: '',
+    },
+    validate: zodResolver(resetPasswordRequestSchema),
+  });
+
+  const successNotification = () => {
+    notifications.show({
+      title: 'Votre requête a bien été prise en compte',
+      message:
+        'Si un compte est associé à cet email, vous recevrez un email de réinitialisation de mot de passe.',
+      autoClose: 5000,
+      color: 'teal',
+      icon: <IconCheck />,
+    });
+  };
+
+  const errorNotification = (message: string) =>
+    notifications.show({
+      title: "Une erreur s'est produite",
+      message: message,
+      autoClose: 2000,
+      color: 'red',
+      icon: <IconX />,
+    });
+
```

Comment:
2s si to short for read, and idk the error message, but maybe he can be longer than 24 characteres,
5s better and u can close it manually if you want so..